site/tests/app/models/UserTester.php

```diff
@@ -114,6 +114,7 @@ public function testToObject() {
             'user_preferred_lastname' => null,
             'user_email' => "test@example.com",
             'user_group' => 1,
+            'user_access_level' => 3,
```

Comment:
Instead of hard coding level three, use the defined access level constant, such as "User::LEVEL_USER". This goes for all places with an access check.